docs/pages/enroll-resources/server-access/guides/encrypted-session-recordings.mdx

```diff
@@ -0,0 +1,269 @@
+at-rest encryption of their session recording data.
+
+## How it works
+
+When encrypted recordings are enabled, Teleport will encrypt session recording
+data before saving it to disk or long term storage. Encryption keys are
+provisioned using the same key storage backend configured for CAs defined by
+the `ca_key_params` section of the Teleport Auth Service configuration file. In
+`node-sync` and `proxy-sync` session recording modes, session recording events
+are sent directly to the Teleport Auth Service where they are encrypted prior
+to being written to long term storage. In `node` and `proxy` session recording
+modes, session recording events are encrypted locally at the node or proxy
+before being written to disk for future upload to the Auth Service.
+
+Decryption of session recordings for replay is always facilitated by the
+Teleport Auth Service regardless of the session recording mode. This means
+replays can be viewed using the Teleport Web UI, or by using `tsh play`
+with a valid session ID. Replaying session recording files directly using
+`tsh play` is not possible for encrypted files.
+
+## Step 1/2. Configure Teleport
+
+### Enable recording encryption
+
+Encrypted session recordings can be enabled through both the Teleport Auth
+Service configuration file or by modifying the `session_recording_config`
+resource dynamically using `tctl`.
+
+You can edit the Auth Service configuration file as follows:
+
+```yaml
+# snippet from teleport.yaml
+auth_service:
+  session_recording_config:
+    encryption:
+      enabled: yes
+```
+
+You can also edit the `session_recording_config` resource using
+`tctl edit session_recording_config`:
+
+```yaml
+# snippet from session_recording_config
+spec:
+  encryption:
+    enabled: yes
+```
+
+If you are already configuring session recording through the configuration file
+it is recommended that you enable encryption there as well, as modifications
+made through `tctl` will be overwritten when the Teleport Auth Service
+restarts.
+
+### Optional step: Configure key storage backend
+
+Session recording encryption keys are provisioned using the same key storage
+backend configured for CAs. Software keys stored in Teleport's backend storage
+are the default, but other backends can also be configured:
+
+- [AWS KMS](../../../zero-trust-access/deploy-a-cluster/aws-kms.mdx)
+- [Google Cloud KMS](../../../zero-trust-access/deploy-a-cluster/gcp-kms.mdx)
+- [HSM](../../../zero-trust-access/deploy-a-cluster/hsm.mdx)
+
+It is important that all Teleport Auth Services have access to the same keys
+and key storage backend to ensure all recorded sessions are always replayable.
+
+## Step 2/2. Confirm new recordings are encrypted
+
+It is possible to confirm that session recordings are now encrypted by
+capturing a new recording and downloading the resulting `.tar` file from your
+[audit sessions backend](../../../reference/architecture/session-recording.mdx#storage).
```

Comment:
We typically advise against including links in the bodies of how-to guides unless following them has a clear objective related to the objective of the guide. In this case, this link does not have a clear objective, so I suggest we remove it.